### PR TITLE
Further SMG Improvements

### PR DESCRIPTION
**Pull request by Surrealaser**
-Wielding and one-handing accuracy and scatter slightly improved.

-Falloff reverted back to -1 per tile traveled.

-Gyro now works with the SMG.

### CODE REVIEW

**Comment by Rohesie** [comment]
No other changes besides the falloff?

**Comment by Surrealaser** [comment]
@Rohesie At this point, yes.

**Comment by Rohesie** [comment]
Document your changes. This is not the only thing the PR modifies.

**Comment by Surrealaser** [comment]
@Rohesie Those are the current SMG values in place on the test merge, or they should be.

File: code/modules/projectiles/updated_projectiles/guns/smgs.dm [hunk 1]
```diff
@@ -60,8 +60,8 @@
 
 /obj/item/weapon/gun/smg/m39/set_gun_config_values()
 	fire_delay = config.low_fire_delay
-	burst_delay = config.mlow_fire_delay
-	burst_amount = config.high_burst_value
+	burst_delay = config.min_fire_delay
+	burst_amount = config.med_burst_value
```
Comment (Rohesie) [inline code/modules/projectiles/updated_projectiles/guns/smgs.dm]: This is an undocumented change.

Comment (Surrealaser) [reply code/modules/projectiles/updated_projectiles/guns/smgs.dm]: As above.

It's per the test merge; you merged the older iteration of this which had same values.

**Comment by Rohesie** [comment]
Perhaps revert the falloff back to the usual 1 and worsen the gun while unwielded, since it gives no wielded slowdown. Then we can repurpose the pistols as viable one-handed weapons.

**Pull request merged**